scout/commands/download/ensembl.py

```diff
@@ -8,9 +8,20 @@
 
 from scout.utils.ensembl_biomart_clients import EnsemblBiomartHandler
 
+CHROM_SEPARATOR = "[success]"
+NR_EXPECTED_CHROMS = 24
+
 LOG = logging.getLogger(__name__)
 
 
+def integrity_check(nr_chromosomes_in_file: int):
+    if nr_chromosomes_in_file < NR_EXPECTED_CHROMS:
+        raise BufferError(
+            "Ensembl resource does not seem to be complete. Please retry downloading the file."
```

Comment:
```suggestion
            "Ensembl resource does not seem to be complete. Please retry downloading genes/transcripts."
```
Perhaps pass the file type, or catch the exception in the processing function where the resource type is known.

Reply:
File name is printed immediately before, it should be clear from the context, I think